### ISSUE TITLE
Taskbar icon does not open the minimized obsidian.

### ISSUE THREAD

**Issue by ItzAmirreza** [dragonwocky/obsidian-tray#14]
I believe the title explains. How to recreate it? (At least in windows)

1. Close the app and have the option which puts the tray icon
2. Now if you have set obsidian app in your taskbar previsouly and click on it, it opens the page that asks which vault to open, not the minimized version of the app that appears when you click on the tray icon.

**Comment by dragonwocky** [comment]
Yes- when the windows are "hidden," they become dissociated from the taskbar pin and clicking the taskbar triggers the opening of a new vault. Definitely something I'll fix at some point, but for now to resume using hidden windows please use the hotkey or click the tray icon instead of the taskbar pin.

If you would prefer your windows to still be accessible from the taskbar (i.e. you only want them "minimised" not "hidden"), you can disable the "run in background" option to achieve that.

**Comment by legolasdimir** [comment]
lol. Was just coming to report this but it looks like someone already did.

**Comment by traptw1thin** [comment]
+1 It is a bit annoying to have to go to the tray every time (or remember the keyboard shortcut). Not a deal breaker but would be nice if it could be fixed.

**Comment by rose-munaretto** [comment]
Hey @dragonwocky, about "when the windows are "hidden," they become dissociated from the taskbar pin", do you already have any ideas on where to start to fix this? I'd love to help with a PR if possible but I'm not sure between the Electron API and your code, where to look to fix this.
